### PR TITLE
Install client only

### PR DESCRIPTION
**Pull request by janhorstmann**
This PR allows the installation of percona mysql client and its configuration without pulling in percona server

### CODE REVIEW

File: percona/config.sls [hunk 1]
```diff
@@ -27,23 +23,18 @@ mysql_python_dep:
...
-    - context:
-        config: {{ content |default({}) }}
-    - require_in:
-      - pkg: percona_client
-      - pkg: percona_server
-    - require:
-      - file: {{ percona_settings.config_directory }}
+{%     if not extend_written %}
+extend:
+{%         set extend_written = true %}
```
Comment (dkorittki) [inline percona/config.sls]: intendation

File: percona/config.sls [hunk 2]
```diff
@@ -27,23 +23,18 @@ mysql_python_dep:
...
-    - template: jinja
-    - context:
-        config: {{ content |default({}) }}
-    - require_in:
-      - pkg: percona_client
-      - pkg: percona_server
-    - require:
-      - file: {{ percona_settings.config_directory }}
+{%     if not extend_written %}
+extend:
```
Comment (dkorittki) [inline percona/config.sls]: Instead of using extend, we should check if `percona.config` has content.

**Review by dkorittki — CHANGES_REQUESTED** [review]

File: percona/config.sls (hunk 1 above)
Comment (janhorstmann) [reply percona/config.sls]: No longer applicable

File: percona/config.sls (hunk 2 above)
Comment (janhorstmann) [reply percona/config.sls]: Done in:  
b02e19bec22865e60a2d85dd82dece512af036d4

**Review by dkorittki — APPROVED** [review]

**Pull request merged**